sydent/http/info.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+# copyright 2019 new vector ltd
```

Comment:
nit: `New Vector Ltd.`

Reply:
Oop, the entire copyright header got lowercase. Probably a vim mispress.